packages/camlidl/camlidl.1.09.1/opam

```diff
@@ -0,0 +1,33 @@
+opam-version: "2.0"
+synopsis: "Stub code generator for OCaml"
+description: """
+CamlIDL is a stub code generator for OCaml. It automates the
+generation of C stub code required for the Caml/C interface, based on
+an MIDL specification. Also provides some support for Microsoft's COM
+software components."""
+maintainer: "Nicolas Berthier <m@nberth.space>"
+authors: ["Xavier Leroy"]
+license: [
+  "QPL-1.0 WITH OCaml-LGPL-linking-exception"
+  "LGPL-2.0-or-later WITH OCaml-LGPL-linking-exception"
+]
+homepage: "https://github.com/xavierleroy/camlidl"
+dev-repo: "git+https://github.com/xavierleroy/camlidl.git"
+bug-reports: "https://github.com/xavierleroy/camlidl/issues"
+build: [
+  ["mv" "config/Makefile.unix" "config/Makefile"] {os != "win32"}
+  ["mv" "config/Makefile.mingw" "config/Makefile"] {os = "win32"}
+  [make "all"]
+]
+depends: [
+  "ocaml" {>= "4.03"}
+]
+extra-files: [
+  ["camlidl.install" "md5=cf56e14faed046880b7c9d0f4cd737f1"]
+  ["META" "md5=f12cc59517db6c176c1ec3809dc12bac"]
+]
+url {
+  src:
+    "https://github.com/kit-ty-kate/camlidl/archive/500.tar.gz"
```

Comment:
Why is my hotfix branch in this package?

Reply:
At the beginning that was mostly to see whether the CI would pass. I looked at the diff w.r.t official 1.09 and thought it was legit enough for a quick release of a minor version.
Is there a branch/commit/PR in @xavierleroy's camlidl repository so we could do an official release for a 1.09-like version for ocaml 5.0?

Reply:
Would would not accept versions that are not official. If upstream does not want/have time to release 1.09.1 then a patch release can be done instead (1.09-1, where the official 1.09 tarball would be used but you can add a patch to the `files` directory) but I'd rather avoid it as a default when possible